Sources/SentryDemangling/lib/Demangling/Punycode.cpp

```diff
@@ -0,0 +1,381 @@
+#pragma clang diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#pragma GCC diagnostic ignored "-Wshorten-64-to-32"
+#pragma GCC diagnostic ignored "-Wshadow"
+//===--- Punycode.cpp - Unicode to Punycode transcoding -------------------===//
+//
+// This source file is part of the Swift.org open source project
```

Comment:
We likely need to add the license to our repo too? @bruno-garcia

Reply:
As a header t ok  the file itself is also ok (it's MIT or Apache. 2?)

Reply:
Apache 2